salesforce-data-cloud/1.0/modules/ROOT/pages/salesforce-data-cloud-connector-xml-maven.adoc

```diff
@@ -0,0 +1,81 @@
+= Salesforce Data Cloud Connector 1.0 - XML and Maven Support - Mule 4
+
+
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio:
+
+* If you manually code a Mule runtime engine (Mule) app in XML either from the Anypoint Studio XML editor or from a text editor, you can access the connector from your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
+* If you add the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location and it also adds a dependency to the `pom.xml` file.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `mule` tag of the header 
+of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/salesforce-data-cloud 
+http://www.mulesoft.org/schema/mule/salesforce-data-cloud/current/mule-salesforce-data-cloud.xsd
+----
+
+This example shows how the namespace statements are placed in the XML: 
+
+[source,xml,linenums]
+----
+<mule xmlns:$2="http://www.mulesoft.org/schema/mule/$2" 
```

Comment:
@valkyrie69 Do we need to replace the $2 with an appropriate value for our connector or is this variable configured somewhere?

Reply:
@adatta0728 Yes, you need to replace that variable with the connector name. I don't know what this connector is named, but it would be something similar to `salesforce-data-cloud`. If you add the connector to a Studio project, you can check the XML and see what it's called.

Reply:
thanks for that guidance, our namespace is sdc. 
**xmlns:sdc="http://www.mulesoft.org/schema/mule/sdc"**
will update it in my copy of changes